portfolio/src/app/case-studies/page.jsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Image from 'next/image'
+
+import { CarRentDesktop } from '../../../assets'
+
+const page = () => {
+  return (
+    <div className=''>
+
+      <div className='flex justify-center py-14 bg-grey-2 dark:bg-grey-5'>
+        <div className='flex flex-col justify-center items-center bg-case-studies-banner bg-no-repeat bg-cover bg-center bg-fixed h-96 p-16 rounded'>
+          <div className='flex flex-col items-center text-center'>
+            <p className='text-white font-bold text-6xl'>Recent Case Studies</p>
+            <p className='text-white font-light text-2xl'>Diving Deep into Our Most Recent Projects: Insights, Challenges and Triumphs</p>
+          </div>
+        </div>
+      </div>
+
+      <div className='flex flex-col py-8'>
+        <div className='flex justify-center gap-4'>
+          
+          <div className='flex-col'>
+            <div className='bg-pink-1 pt-8 rounded'>
+              <div className='px-6'>
+                <Image src={CarRentDesktop} />
```

Comment:
You should always include 'width' and 'height' attributes in the Next.js Image component. These are required for preventing Cumulative Layout Shift (CLS), an important web vitals metric for Google SEO.

Here is a link if you want to read more about it. 
https://refine.dev/blog/using-next-image/#introduction